generators/client/templates/angular/src/main/webapp/app/shared/tracker/_tracker.service.ts

```diff
@@ -11,6 +10,9 @@ import { CSRFService } from '../auth/csrf.service';
 import { AuthServerProvider } from '../auth/auth-jwt.service';
 <%_ } _%>
 
+import SockJS = require('sockjs-client');
```

Comment:
Hi @gzsombor I was wondering why sockjs and stomp were chosen for this rather than [Observable.websocket](http://reactivex.io/rxjs/class/es6/Observable.js~Observable.html#static-method-webSocket). Thanks.